core/src/cluster_info.rs

```diff
@@ -361,51 +365,59 @@ impl ClusterInfo {
         let now = timestamp();
         let mut spy_nodes = 0;
         let mut archivers = 0;
+        let mut different_shred_nodes = 0;
```

Comment:
`contact_info_trace()` changes filters out nodes of different shreds for more concise view of the real network